sphinx_docs/conf.py

```diff
@@ -0,0 +1,37 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+
+import sys
+import os
+
+sys.path.insert(0, os.path.abspath("../src"))
+
+project = "Groundlight Python SDK"
+copyright = "2023, Groundlight AI <support@groundlight.ai>"
+author = "Groundlight AI <support@groundlight.ai>"
+release = "0.11.1"
```

Comment:
We need to make sure this gets automatically updated with the current release version.  There's some trick which sets `groundlight.__version__` to the release number - we should use the same kind of thing here.  Otherwise this will get out of sync.